v-next/example-project/hardhat.config.ts

```diff
@@ -95,7 +95,7 @@ const greeting = task("hello", "Prints a greeting")
   })
   .build();
 
-export default {
```

Comment:
I believe there was a reason why we used `satisfies` here, but I can't remember why. @alcuadrado do you remember?

Reply:
I think this is the right move. We were using `satisfies` instead of `as`, which is much much better, but `: T` is equivalent to `satisfies` in this case.